src/seedu/addressbook/data/person/Name.java

```diff
@@ -55,6 +56,37 @@ public boolean equals(Object other) {
                 || (other instanceof Name // instanceof handles nulls
                 && this.fullName.equals(((Name) other).fullName)); // state check
     }
+    
+    public boolean isSimilar(Name other){
+    	String myName = this.fullName.toLowerCase();
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.